src/libraries/Microsoft.Extensions.Caching.Memory/src/MemoryCache.cs

```diff
@@ -329,7 +329,7 @@ private bool UpdateCacheSizeExceedsCapacity(CacheEntry entry, CoherentState cohe
                 return false;
             }
 
-            long newSize = 0L;
+            long newSize;
```

Comment:
This can be declared inside the loop where it's first assigned.

Reply:
Done